src/Dialog/Dialog.js

```diff
@@ -105,6 +107,8 @@ function getStyles(props, context) {
       transition: open ?
         transitions.easeOut('0ms', 'left', '0ms') :
         transitions.easeOut('0ms', 'left', '450ms'),
+      direction: isRtl ? 'rtl' : 'ltr',
```

Comment:
That's raising a broad issue. Should each component handle the direction or we should we ask users to apply it at the root of the DOM, like the `body` element?